pkg/mkimage-raw-efi/install

```diff
@@ -265,11 +268,21 @@ DISK_WITH_P3=""
 DISKS_TO_MERGE_COUNT=0
 MULTIPLE_DISKS=false
```

Comment:
Are these variables still being used or can we delete them entirely?

Reply:
They are used in following cases:
MULTIPLE_DISKS is really used for logging to installer.log where we will know if persist was installed on multiple disks.
DISKS_TO_MERGE_COUNT is useful to check zfs minimum requirements checks. Say install was performed just provided multiple disks (like old way of doing zfs installs) but without explicit eve_install_zfs, then we need to check for disks count.